### PR TITLE
Add --ie32 commandline option to webdriver_manager

### LINKED ISSUE #1297: [Feature request][webdriver-manager] Allow a command-line switch to get 32bit browser driver(s)
The 64bit IE driver is horrendously bad when used with recent versions of IE - see selenium's [bug tracker](https://code.google.com/p/selenium/issues/detail?id=5116).

Currently, if I want to test IE with protractor, I need to run `node webdriver-manager update`, then manually download the 32bit IE driver. Currently, there is the `--ie` switch for webdriver-manager, but it will _always_ get the 64bit IE driver if your system architecture is 64bit. It would be pretty nice to specify `--ie32` or something to automatically download the 32bit IE driver

### PR DESCRIPTION
**Pull request by rogierschouten**
Fixes #1297

The new option allows to download the 32-bit version of the IE driver on
a 64-bit system, as the 64-bit version has been broken for over a year
now (the sendKeys() function works very slowly on it).

### CODE REVIEW

**Comment by rogierschouten** [comment]
CLA has been signed in the meantime

**Comment by rogierschouten** [comment]
Commit author has been updated to match the CLA

File: bin/webdriver-manager [hunk 1]
```diff
@@ -92,7 +106,9 @@ var cli = optimist.
     describe('ignore_ssl', 'Ignore SSL certificates').boolean('ignore_ssl').
     default('ignore_ssl', false).
     describe('proxy', 'Proxy to use for the install or update command').
-    describe('alternate_cdn', 'Alternate CDN to the binaries');
+    describe('alternate_cdn', 'Alternate CDN to the binaries').
+    describe('ie32', 'Use 32-bit IE driver regardless of current architecture').boolean('ie32').
```
Comment (hankduan) [inline bin/webdriver-manager]: This doesn't do anything. The line `for (bin in binaries) {...` actually adds all the instructions for the binary installations.

Comment (hankduan) [inline bin/webdriver-manager]: This doesn't do anything. The line `for (bin in binaries) {...` actually adds all the instructions for the binary installations.

Comment (hankduan) [inline bin/webdriver-manager]: Maybe change the name of the binary to 'IEDriver-32bit', and remove the describe statement.

**Comment by hankduan** [comment]
@rogierschouten Have you tested it? I can't test it right now since I don't have a windows machine, but the code looks good. As soon as you can confirm that you tested it, and sign the CLA again, I'll pull it in.

**Comment by rogierschouten** [comment]
I plan to test it at work next Monday, I can't do so now either. I'll let you know, thank you

**Comment by rogierschouten** [comment]
Tested & working!

**Comment by hankduan** [comment]
Thanks! Merged: https://github.com/angular/protractor/commit/c989a7eeed5a0a55d2fbd37dc7278a7967889852

**Pull request closed without merging**